create-release.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+
+const { Octokit } = require('@octokit/rest');
+const inquirer = require('inquirer');
+
+const octokit = new Octokit({
+    auth: '<GENERATE FROM HERE: https://github.com/settings/tokens>',
```

Comment:
OK, so token goes here! Is there a security risk to putting a token here? Sorry, just to be sure!

Reply:
Ah @Sagarpreet - check it out - here it says to put the token in the "secrets" --- where it can be referenced from:

https://riggaroo.dev/using-github-actions-to-automate-our-release-process/

also here: https://github.com/actions/create-release#example-workflow---create-a-release

Let me see about that...

Reply:
aha yes https://github.com/publiclab/PublicLab.Editor/settings/secrets - doing this now!

Reply:
Would this work then? 

```suggestion
    auth: ${{ secrets.GITHUB_TOKEN }},
```

Reply:
OK, adding these privileges: 
![image](https://user-images.githubusercontent.com/24359/98267387-55627b80-1f59-11eb-8160-fd540f932732.png)

Reply:
OK, strangely I couldn't make a secret called `GITHUB_TOKEN` as it said it was an invalid name. I instead made `GITHUBTOKEN`. I wonder... is there an organization secret  with a colliding name? Not sure, but let's try `GITHUBTOKEN`

Reply:
Just noting that we have no org secrets so it wasn't a name collision...

oh! Aha - it's a reserved name -- it already exists with some permissions. Maybe best try it out? https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

i wasn't sure it had release creation permissions. but if it does, might as well use it instead of the one I created.

Reply:
So I think anyone can create a release in open source projects, right? 
Similar to issues maybe, not sure.
So only read permissions for projects should be enough